devsetup/Makefile

```diff
@@ -97,11 +98,27 @@ edpm_compute_repos: ## Configure EDPM compute VM with repositories
 	$(eval $(call vars))
 	scripts/edpm-compute-repos.sh ${EDPM_COMPUTE_SUFFIX}
```

Comment:
I'd then add the loop here as well so that you don't need to call this target multiple times.